docs/integrations/generic-commands-about.md

```diff
@@ -0,0 +1,14 @@
+---
+id: generic-commands-about
+title: Generic Commands
+---
+
+
+## About Generic Commands
+
+Today the current XSOAR platform supports over 450+ integrations. 
+Some commands can be generalized across similar integrations. It allows the  
+These commands can be used both on all integrations or with `using` parameter on a specific instance of an integration.  
```

Comment:
Can you be a little more specific in describing what these commands are and why are important, maybe with some examples?

Also is `using` documented somewhere in the product doc you can link to?

Reply:
no we don't afaik